plugin/pkg/auth/authorizer/rbac/bootstrappolicy/policy.go

```diff
@@ -177,6 +177,10 @@ func ClusterRoles() []rbac.ClusterRole {
 					"/swagger.json", "/swagger-2.0.0.pb-v1",
 					"/api", "/api/*",
 					"/apis", "/apis/*",
+					"/bulk", "/bulk/*",
```

Comment:
I'm pretty sure bulk should not be a top-level path. Have you thought of any alternatives?